### PR TITLE
ARGO-372 Document Changes for supporting Multiple Monitoring Engines …

### PR DESCRIPTION
**Pull request by kaggis**
…in ARGO Compute Engine

- [x] added .md doc in `/docs/features/001-multiple-monitoring-engines`

### CODE REVIEW

File: doc/features/001-multiple-monitoring-engines.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+This is an RFC for ARGO Compute Engine to support metric data received from two
+or more monitoring engines enabled in active-active mode
+
+# Motivation
+
+Till now ARGO Compute Engine received metric data from a distributed monitoring
+engine involving Nagios nodes dedicated for each high-level group.
+
+With the introduction of a Centralized Single Monitoring Engine accompanied with
+the an HA plan to run two of them in active-active setup, Compute Engine needs to
```
Comment (themiszamani) [inline doc/features/001-multiple-monitoring-engines.md]: Engine accompanied with the an HΑ plan --> with the HA plan

**Comment by pkoro** [comment]
+1

**Comment by dpavlos** [comment]
+1

**Comment by themiszamani** [comment]
test this please

**Comment by themiszamani** [comment]
test this please

**Comment by themiszamani** [comment]
test this please

**Comment by themiszamani** [comment]
test this please

**Comment by themiszamani** [comment]
test this please

**Pull request merged**